src/views/Main/Main.js

```diff
@@ -0,0 +1,54 @@
+import { fetchByType, fetchPokemon, fetchType } from '../../services/pokemon';
+import { usePokeContex } from '../../context/PokeContext';
+import Search from '../../components/Search';
+
+import './Main.css';
+
+import React, { useEffect, useState } from 'react';
+import PokeList from '../../components/PokeList';
+
+export default function Main() {
+  // const { selectedType, pokemon, setPokemon, search, order, setTypes } = usePokeContex();
+  const [pokemon, setPokemon] = useState([]);
+  const [selectedType, setSelectedType] = useState('All');
+  const [search, setSearch] = useState('');
+  const [order, setOrder] = useState('asc');
+  const [types, setTypes] = useState([]);
+  const [load, setLoad] = useState(false);
+
+  useEffect(() => {
+    const allPokemon = async () => {
+      const everyPokemon = await fetchPokemon();
+      setPokemon(everyPokemon);
+      const pokeTypes = await fetchType();
+      setTypes(['All', ...pokeTypes]);
+
+      setTimeout(() => {
+        setLoad(false);
+      }, 2000);
```

Comment:
Try modifying or removing the timeout in order to get rid of some testing errors:

const timer = setTimeout(() => {
          setLoading(false);
        }, 3000);
        return () => {
          clearTimeout(timer);
        };